lib/parseTokenList.js

```diff
@@ -7,10 +7,12 @@
  * @private
  */
 
-function parseTokenList (str) {
+const slice = String.prototype.slice
```

Comment:
we destructure slice and not invoke instance method?

Reply:
slice is ambigous for node. Is it slice of string or slice of array. so it is potential deoptimization issue. So to make it easier for node, you can define them in variable and call them directly.